lib/decidim/component_manager.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
+module Decidim
+  class ComponentManager
```

Comment:
Docs?

Reply:
Thanks @mrcasals, I'll add some comments. We seem to be blindly requiring all class definitions to be commented, should we enable 
https://github.com/decidim/decidim/blob/80508ca5bc6197fa1d8653ab268dd43849837551/.rubocop.yml#L802-L804?

Reply:
Happy fiestas, by the way! :smile:

Reply:
@deivid-rodriguez sorry for the late reply. I like having class comments to explain the purpose of the class. Sometimes it's a dull comment (for example, most of the controllers), but for other classes I find it useful to have a little context on why use that class and how to do it, so I would not disable this check.

Hope you had some happy fiestas you too!

Reply:
Maybe we can enable the check with folder exceptions such as `controllers/`.